packages/jest-runner/worker.ts

```diff
@@ -46,6 +54,7 @@ type JazzerTestResult = {
 export class JazzerWorker {
 	static #workerInitialized = false;
 	static #currentTestPath = "";
+	readonly defaultTimeout = 5000;
```

Comment:
Is the value here based on experience/arbitrary/...? Maybe a comment would be helpful

Reply:
It's the default Jest timeout, added a comment.